tests/LoopDecoratorTest.php

```diff
@@ -310,8 +310,8 @@ public function testFutureTick()
             $called['futureTickTick'] = true;
         });
 
-        Phake::when($loop)->futureTick($listener)->thenReturnCallback(function ($listener) {
-            $listener();
+        Phake::when($loop)->futureTick($listener)->thenReturnCallback(function ($listener) use ($loop) {
+            $listener($loop);
```

Comment:
I don't know if I can make that meaningful. `$loop` already comes from the outer context. It's not being passed in with the argument.